pkgs/applications/networking/whoogle-search/default.nix

```
@@ -0,0 +1,78 @@
+{ lib
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "whoogle-search";
+  version = "0.7.1";
+
+  src = python3.pkgs.fetchPypi {
+    inherit pname version;
+    hash = "sha256-jn5Plm0yps3yww9Uj3Z77HejuAEMhw1/MqJpCcn7Vv8=";
+  };
+
+  postPatch = ''
+    # Remove the version pinning
+    sed -i 's/[><=]=[0-9.]*//' requirements.txt
+
+    # Remove dev packages and packages installed by dependencies
```

```suggestion
    # Remove dev packages
```

We shouldn't rely on that and they cause no harm. Only dev things should be removed.

---

I have added all the requirements from `setup.cfg`: https://github.com/benbusby/whoogle-search/blob/main/setup.cfg#L19=